tests/cnf/core/network/metallb/tests/common.go

```diff
@@ -284,3 +285,28 @@ func metalLbDaemonSetShouldMatchConditionAndBeInReadyState(
 	}, tsparams.DefaultTimeout, tsparams.DefaultRetryInterval).Should(expectedCondition, errorMessage)
 	Expect(metalLbDs.IsReady(120*time.Second)).To(BeTrue(), "MetalLb daemonSet is not Ready")
 }
+
+func createBGPPeerWithConnectTimeAndVerifyIfItsReady(connectTime metav1.Duration,
```

Comment:
remove  this func and use createBGPPeerAndVerifyIfItsReady(). Just add additiona atribute ConnectTime and if it's not 0 run bgpPeer.WithConnectTime(connectTime)

Reply:
removed and updated